app/screens/ErrorScreen.tsx

```diff
@@ -0,0 +1,57 @@
+import { useNavigation, useRoute } from "@react-navigation/native";
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { View } from "react-native";
+import AppLayout from "../components/AppLayout";
+import Routes from "../config/Routes";
+import { H1, H3 } from "../elements/Texts";
+import AppStyles from "../styles/AppStyles";
+
+export enum ErrorScreenType {
+  LOGIN_FAILED,
+}
+
+const ErrorScreen = () => {
+  const route = useRoute();
+  const nav = useNavigation();
+  const { t } = useTranslation();
+  const [screenType, setScreenType] = useState<ErrorScreenType>();
+
+  const params = route.params as any;
+
+  useEffect(() => {
+    setScreenType(params?.screenType as ErrorScreenType);
+    nav.navigate(Routes.Error, {
```

Comment:
Just for understanding. Why the reload if you are already here?

Reply:
We have this on payment for every screen, where we pass some arguments to hide them from users. For example the same is done on /login, /kyc or /link